addons/scratchstats/addon.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "Profile statistics",
-  "description": "Shows statistics from ScratchDB on user profiles.",
+  "description": "Shows statistics on user profiles.",
```

Comment:
```suggestion
  "description": "Shows statistics from ScratchDB on user profiles.",
```

Reply:
Probably because he added it on the credits? I don't know, but it's better to expose it better.